set.mm

```diff
@@ -415270,9 +415352,9 @@ the walk is closed if v(n) = v0".
        represented by the following sequence: p(0) e(f(0)) p(1) e(f(1)) ...
        p(n-1) e(f(n-1)) p(n)=p(0).
 
-       Notice that by this definition, a single vertex is a closed walk of
-       length 0, see also ~ 0clwlk !  (Contributed by Alexander van der Vekens,
-       12-Mar-2018.)  (Revised by AV, 16-Feb-2021.) $)
+       Notice that by this definition, a single vertex can be considered as a
+       closed walk of length 0, see also ~ 0clwlk .  (Contributed by Alexander
+       van der Vekens, 12-Mar-2018.)  (Revised by AV, 16-Feb-2021.) $)
     df-clwlks $a |- ClWalks = ( g e. _V |-> { <. f , p >. |
                  ( f ( Walks ` g ) p /\ ( p ` 0 ) = ( p ` ( # ` f ) ) ) } ) $.
```

Comment:
Maybe some confusion come from the different definitions in the literature.
In the comment, you write that the definition of Huneke for a closed walk is a sequence of vertices, however the definition we use for walks is the one from Bollobas, which is an alternating sequence of vertices and edges... therefore the different properties, I assume?

Reply:
See my response to the general comment.